[Scout] Align skills and docs with current Scout behavior (#281056)
elastic/kibana: .agents/skills/scout-migrate-from-ftr/references/generate-plan.md

1. **Catalog every FTR service and page object** used by the tests: name, what it does, which files use it, and whether it contains hidden assertions (`existOrFail`, `missingOrFail`, or `expect` calls that should move to specs)
2. **Check for existing Scout equivalents**: does a matching page object or API service already exist in the Scout packages or in other plugins' `test/scout` trees? Note: exists / exists-but-in-wrong-scope / missing
3. **For missing equivalents**, recommend scope (shared Scout package vs solution-scoped vs plugin-local) based on how many plugins would benefit

> 4. **Catalog EUI component interactions**: list every EUI component the tests interact with directly (combo boxes, data grids, selectable lists, etc.) so the executor can use a published Component Object through `page.components.*`, or identify where an existing `Eui*Wrapper` class is temporarily required because no equivalent Component Object exists

Reviewer: ```suggestion
4. **Catalog EUI component interactions**: list every EUI component the tests interact with directly (combo boxes, data grids, selectable lists, etc.) so the executor can use a published EUI test helpers through `page.components.*`.
```
